src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -337,29 +333,13 @@ def _execute_step(
         Raises:
             DuplicateRunNameError: If the run name is already in use.
         """
-        step_name_param = (
-            INTERNAL_EXECUTION_PARAMETER_PREFIX + PARAM_PIPELINE_PARAMETER_NAME
-        )
         pipeline_step_name = tfx_launcher._pipeline_node.node_info.id
         start_time = time.time()
         logger.info(f"Step `{pipeline_step_name}` has started.")
         try:
             execution_info = tfx_launcher.launch()
-
             if execution_info and get_cache_status(execution_info):
-                if execution_info.exec_properties:
```

Comment:
Alright makes sense, I think tfx in some version updated their handling of duplicate runs so that's probably why. But agreed not anything important so I guess removing is fine 👍